Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,19 @@
 import React from 'react';
 import './Card.css';
+
+import CardBanner from './CardBanner';
+import CardContent from './CardContent';
+
+function CardContainer() {
+    return (
+        <a href='https://www.reactjs.org'  target='_blank'>
```

Comment:
Great solution to this. I also like how you've organized your code it's very readable.